dotcom-rendering/src/web/components/ArticleHeadline.tsx

```diff
@@ -287,12 +290,85 @@ const WithAgeWarning = ({
 	return <>{children}</>;
 };
 
+const decideBottomPadding = ({
+	format,
+	hasStarRating,
+	hasAvatar,
+}: {
+	format: ArticleFormat;
+	hasStarRating?: boolean;
+	hasAvatar?: boolean;
+}) => {
+	const defaultPadding = css`
+		padding-bottom: ${space[6]}px;
+		${from.tablet} {
+			padding-bottom: ${space[9]}px;
+		}
+	`;
+	switch (format.display) {
```

Comment:
This switch/case is more complicated because previously not all uses of an ArticleHeadline were wrapped with an ArticleHeadlinePadding?

Reply:
Exactly. Previously there were something like 4 ways to decide padding, all with their own captured logic. Now we are bringing this together into one location - this switch statement.